examples/cra-ts-kitchen-sink/src/react-app-env.d.ts

```diff
@@ -1 +1 @@
-/// <reference types="react-scripts" />
+// / <reference types="react-scripts" />
```

Comment:
⚠️ Nop, this is not a comment but a [triple slash directive](https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html).